test/functional/feature_fee_estimation.py

```diff
@@ -4,7 +4,7 @@
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 """Test fee estimation code."""
 from copy import deepcopy
```

Comment:
There is still a single fee calculation in `feature_fee_estimation.py` done with float in `transact_and_mine` method.

Reply:
added it  here c96568e4e812ad59ae7c922dac3cea0ee880341c